scripts/export_reduced.py

```diff
@@ -21,6 +21,12 @@ def crop_and_shift(hdu, centerpix, crop=200):
     return hdu
 
 
+def reduce_lbt(hdulist):
+    for i in range(1, 5):
+        x, y = hdulist[i].header["NAXIS1"], hdulist[i].header["NAXIS2"]
+        hdulist[i] = crop_and_shift(hdulist[i], (int(x/2), int(y/2)), crop = 100)
+    return hdulist
+
```

Comment:
Looks ok to me, but you should add another `elif` down at the line numbers [114-128 ](https://github.com/dirac-institute/trailblazer-testdata/pull/3/files#diff-e81f4743b94ce87af25f2eec71a23059254bae2f1021abf42981d48f757f6371R114) otherwise the function will never run.